### PR TITLE
feat: add Jest Configurator to the Tealium package

### PR DESCRIPTION
**Pull request by jimhunty**
As this only is used on web, I have only added the web config.

Also there was no coverage for `tealium.js` and `storybook.js`. Does this need to be added?

### CODE REVIEW

File: packages/tealium/package.json [hunk 1]
```diff
@@ -18,31 +18,10 @@
...
-    ],
-    "setupFiles": [
-      "raf/polyfill"
-    ]
-  },
   "dependencies": {
     "@storybook/addon-actions": "3.3.11",
-    "@storybook/addons": "3.3.11"
+    "@storybook/addons": "3.3.11",
+    "@times-components/jest-configurator": "0.0.23"
```
Comment (colincclark) [inline packages/tealium/package.json]: should be `devDependencies`

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/dd5697eaa2fc8cb025d7edc31d3c1394bfcfacaf/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f51b339c7c71876ebb795aade2ede94499cf7fe6/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/8159735783cff2f6c614e8e54d86ba37d3978572/dextrose-test/index.html

**Pull request merged**